src/main/java/org/fundacionjala/at15/pokemon/TrainerBattle.java

```diff
@@ -1,9 +1,68 @@
 package org.fundacionjala.at15.pokemon;
 
 public class TrainerBattle extends Battle {
+    private Trainer trainer1;
+    private Trainer trainer2;
+    private boolean battleOver = false;
+    private int turn = 0;
+    private final int money = 50;
+
+    public TrainerBattle(Trainer newTrainer1, Trainer newTrainer2) {
+        this.trainer1 = newTrainer1;
+        this.trainer2 = newTrainer2;
+    }
+
+    //get pokemon available starting with the first (0)
+    public Pokemon getPokemonAvailable(Trainer trainer) {
+        int currentHitPoints;
+        PokemonTeam pokemonTeam = trainer.getPokemonTeam();
+        Pokemon currentPokemon = pokemonTeam.getPokemonTeam().get(0);
```

Comment:
I think this line is unnecessary, because you are getting the same pokemon inside 'for' bucle. Maybe you can only declare currentPokemon object without instancing.

Reply:
Yes but i think it's necesary to return something